src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -230,7 +232,56 @@ const formConfig = {
         },
         // pageFour: {},
         // pageFive: {},
-        // pageSix: {},
+        privateRecordChoice: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-choice',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          depends: (formData, index) => _.get(`disabilities.${index}.view:privateMedicalRecords`, formData),
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:title': disabilityNameTitle,
+                'ui:description': privateRecordsChoice,
+                'view:uploadPrivateRecords': {
+                  'ui:title': 'Do you want to upload your private medical records?',
+                  'ui:widget': 'radio',
+                  'ui:options': {
+                    labels: {
+                      yes: 'Yes',
+                      no: 'No, please get them from my doctor'
+                    }
+                  }
+                },
+                'view:privateRecordsChoiceHelp': {
+                  'ui:description': privateRecordsChoiceHelp
+                }
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    'view:uploadPrivateRecords': {
```

Comment:
Would it make sense to create a component for this rather than include an object without properties (`view:privateRecordsChoiceHelp`)?